src/hardware/input/mouseif_virtual_machines.cpp

```diff
@@ -45,8 +45,12 @@ static struct {
 	float delta_wheel = 0.0f;   // accumulated mouse wheel movement
 } vmware;
 
-static bool use_relative = true; // true = ignore absolute mouse position, use relative
-static bool is_input_raw = true; // true = no host mouse acceleration pre-applied
+// true = ignore absolute mouse position, use relative
+static bool use_relative = true;
```

Comment:
Alternatively, introduce an enum with the values `Relative` and `Absolute`.

Reply:
I would prefer not to introduce enum values to replace bools - we have a lot of boolean flags, I'm afraid that would have made the code less readable.